ts/webworker/workers/browser/libsession_worker_interface.ts

```diff
@@ -333,6 +350,155 @@ export const ConvoInfoVolatileWrapperActions: ConvoInfoVolatileWrapperActionsCal
+    callLibSessionWorker([`MetaGroupConfig-${groupPk}`, 'memberSetAccepted', pubkeyHex]) as Promise<
+      ReturnType<MetaGroupWrapperActionsCalls['memberSetAccepted']>
+    >,
+  memberSetPromoted: async (groupPk: GroupPubkeyType, pubkeyHex: string, failed: boolean) =>
+    callLibSessionWorker([
+      `MetaGroupConfig-${groupPk}`,
+      'memberSetPromoted',
+      pubkeyHex,
+      failed,
+    ]) as Promise<ReturnType<MetaGroupWrapperActionsCalls['memberSetPromoted']>>,
+  memberSetInvited: async (groupPk: GroupPubkeyType, pubkeyHex: string, failed: boolean) =>
+    callLibSessionWorker([
+      `MetaGroupConfig-${groupPk}`,
+      'memberSetInvited',
+      pubkeyHex,
+      failed,
+    ]) as Promise<ReturnType<MetaGroupWrapperActionsCalls['memberSetInvited']>>,
+  memberSetName: async (groupPk: GroupPubkeyType, pubkeyHex: string, name: string) =>
+    callLibSessionWorker([
+      `MetaGroupConfig-${groupPk}`,
+      'memberSetName',
+      pubkeyHex,
+      name,
+    ]) as Promise<ReturnType<MetaGroupWrapperActionsCalls['memberSetName']>>,
+  memberSetProfilePicture: async (
+    groupPk: GroupPubkeyType,
+    pubkeyHex: string,
+    profilePicture: ProfilePicture
+  ) =>
+    callLibSessionWorker([
+      `MetaGroupConfig-${groupPk}`,
+      'memberSetProfilePicture',
+      pubkeyHex,
+      profilePicture,
+    ]) as Promise<ReturnType<MetaGroupWrapperActionsCalls['memberSetProfilePicture']>>,
+
+  /** GroupKeys wrapper specific actions */
+
+  keyRekey: async (groupPk: GroupPubkeyType) =>
+    callLibSessionWorker([`MetaGroupConfig-${groupPk}`, 'keyRekey']) as Promise<
+      ReturnType<MetaGroupWrapperActionsCalls['keyRekey']>
+    >,
+  keysNeedsRekey: async (groupPk: GroupPubkeyType) =>
+    callLibSessionWorker([`MetaGroupConfig-${groupPk}`, 'keysNeedsRekey']) as Promise<
+      ReturnType<MetaGroupWrapperActionsCalls['keysNeedsRekey']>
+    >,
+  currentHashes: async (groupPk: GroupPubkeyType) =>
+    callLibSessionWorker([`MetaGroupConfig-${groupPk}`, 'currentHashes']) as Promise<
+      ReturnType<MetaGroupWrapperActionsCalls['currentHashes']>
+    >,
+
+  loadKeyMessage: async (
+    groupPk: GroupPubkeyType,
+    hash: string,
+    data: Uint8Array,
+    timestampMs: number
+  ) =>
+    callLibSessionWorker([
+      `MetaGroupConfig-${groupPk}`,
+      'loadKeyMessage',
+      hash,
+      data,
+      timestampMs,
+    ]) as Promise<ReturnType<MetaGroupWrapperActionsCalls['loadKeyMessage']>>,
+  encryptMessage: async (groupPk: GroupPubkeyType, plaintext: Uint8Array) =>
+    callLibSessionWorker([`MetaGroupConfig-${groupPk}`, 'encryptMessage', plaintext]) as Promise<
+      ReturnType<MetaGroupWrapperActionsCalls['encryptMessage']>
+    >,
+  decryptMessage: async (groupPk: GroupPubkeyType, ciphertext: Uint8Array) =>
+    callLibSessionWorker([`MetaGroupConfig-${groupPk}`, 'decryptMessage', ciphertext]) as Promise<
+      ReturnType<MetaGroupWrapperActionsCalls['decryptMessage']>
+    >,
```

Comment:
I think we might need to start breaking this up into smaller files?

Reply:
we just need to build a higer level type which can build all of those for us, rather than having to duplicate the code that much!